testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -59,6 +59,12 @@ pub type RunLoopCounter = Arc<AtomicU64>;
 #[cfg(not(test))]
 pub type RunLoopCounter = ();
 
+#[cfg(test)]
+const UNCONDITIONAL_CHAIN_LIVENESS_CHECK: u64 = 30;
+
+#[cfg(not(test))]
+const UNCONDITIONAL_CHAIN_LIVENESS_CHECK: u64 = 300;
```

Comment:
What is this const controlling? Can you add a rustdoc?

Reply:
There's a thread that periodically wakes up, loads all the affirmation maps, and if it finds a divergence between any of them, will either wake up the coordinator to announce a burn block, or a burn block plus a stacks block.